swagger/src/main/java/grpcbridge/swagger/PathBuilder.java

```diff
@@ -0,0 +1,37 @@
+package grpcbridge.swagger;
+
+import com.google.protobuf.Descriptors;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+
+import java.util.List;
+
+public class PathBuilder {
```

Comment:
Not sure "Builder" makes sense here as nothing is mutated. Would this make more sense as a PathFactory or PathFormatter with a single static method?